docs/how_to/how_to_setup_RO_config_options.rst

```diff
@@ -6,7 +6,7 @@ contains options that account for concentration polarization, the mass transfer
 Example: Configure the RO model to account for concentration polarization and pressure drop
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: python
+.. testcode::
```

Comment:
@bknueven When should we use the `testcode` directive vs. the `testsetup` directive? I noticed that in `how_to_scale_a_model.rst`, `testsetup` was used for a portion of code similar (with a few more lines of code) to this code block.

Reply:
The `testsetup` directive runs, but doesn't display the code, for just the situation where you don't want the user to see the setup to focus on the task being described.

The `testcode` directive displays the code in the documentation, so I replaced `code-block:: python` everywhere I found it with `testcode` so the look of documentation didn't change with this PR. I don't mind the additional imports in the RO config, but they could be done away with as you suggest.

Reply:
Thanks for the explanation! 

I think in this particular case, it is useful to show the extra imports needed for the RO config to show the user how to configure RO to account for concentration polarization and pressure drop. So, I don't think you should do away with them here.